docs/how-to-deploy-cp4waiops-customsize.md

```diff
@@ -0,0 +1,353 @@
+| ------------------------------ | ------------- |--------------------------|
+| cp4waiops.profile              | x-small-idle  | x-small are custom sizing|
+| cp4waiops.eventManager.enabled |false          | not install eventmanager |
+
+
+#### Install *x-small-custom* custom sizing CP4WAIOps 
+| Field                          | Value                   | Description|
+| ------------------------------ | ------------- |--------------------------|
+| cp4waiops.profile              | x-small-custom| x-small are custom sizing|
+| cp4waiops.eventManager.enabled |false          | not install eventmanager |
+
+
+#### Install custom sizing CP4WAIOps in SNO
+
+| Field                          | Value         |  
+| ------------------------------ | ------------- | 
+| cp4waiops.profile              | x-small or x-small-idle  | 
+| cp4waiops.eventManager.enabled |false          |  
+
+And adding following YAML snippet to `HELM` > `VALUES` field will install AI Manager and Event Manager using custom imageCatalog and channel:
+
+```yaml
+cp4waiops:
+  isSNO:true
+```
+
+
+#### Install custom sizing CP4WAIOps using *Custom Build*
+
+The all-in-one configuration also allows you to install CP4WAIOps using custom build by providing specific image catalog and channel.
+
+Just use the install parameters listed in following table when you create the Argo CD App:
+
+| Parameter                           | Type   | Default Value                             | Description 
+| ----------------------------------- |--------|-------------------------------------------|-----------------------------------
+| cp4waiops.aiManager.imageCatalog    | string | icr.io/cpopen/ibm-operator-catalog:latest | The image catalog for AI Manager.
+| cp4waiops.aiManager.channel         | string | v3.4                                      | The subscription channel for AI Manager.
+
+
+These parameters are invisible when you create the Argo CD App from UI, but you can add them when filling in the form in `HELM` > `VALUES` field.
+
+For example, adding following YAML snippet to `HELM` > `VALUES` field will install AI Manager and Event Manager using custom imageCatalog and channel:
+
+```yaml
+cp4waiops:
+  imageCatalog: <my_custom_image_catalog_for_ai_manager>
+  aiManager:  
+    channel: <my_custom_channel_for_ai_manager>
+```
+
+### Verify CP4WAIOps Installation
+
+After both Ceph and CP4WAIOps are ready, you will be able to see those Apps from Argo CD UI as follows with status as `Healthy` and `Synced`.
+
+![w](images/gitops-x-small-idle.png)
+
+![w](images/gitops-x-small-idle-allin1.png)
+
+You can check the topology of CP4WAIOps using Argo CD UI as follows:
+
+![w](images/aimanager-33.png)
+
+You can also check from command line as follows, and make sure there are no error pods. If there are some pods in error state, you can check logs either from Argo CD UI or using `kubectl logs` from command line.
+
+For example, to check custom sizing configmap of AI Manager:
```

Comment:
Shall we expose the custom sizing configmap as the implementation details to readers? If yes, it would be better to explain it at the very beginning of this doc, e.g.: to tell readers that the custom sizing is achieved by applying a special config map that includes all custom sizing settings to different components and will be respected by installer to propagate and enforce these settings down to each component. (If this has been documented officially in IBM Docs, then maybe we can refer to the link simply.) What we did w/ this gitops repo is to store some sample config maps that have been verified to be working to demonstrate the use of this config map.

Reply:
Agree that it is better not to expose the tech details of this configmap, after all, customer does not care how custom sizing happens, but only care to be able to deploy it in their resource limited cluster.